src/accounts.js

```diff
@@ -1,4 +1,25 @@
-const accounts = [];
+const accounts = [
```

Comment:
I guess these codes are just for testing purpose.
Please revert these codes. We are not allowed to modify server codes.
(Only for testing purpose in your local computer we can modify server codes.)

Reply:
Since you deleted accounts variable, I guess your server doesn't work right now.
Please revert this file as it was so that no change will be added to development branch from this branch.
To check this please look "Files changed" tab and make sure that this file is not listed up.